Example/Example/SeriesDetailCollectionViewDataSource.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  SeriesDetailCollectionViewDataSource.swift
+//  Example
+//
+//  Created by Pedro Vicente Gomez on 23/12/15.
+//  Copyright © 2015 GoKarumi S.L. All rights reserved.
+//
+
+import Foundation
+import BothamUI
+
+class SeriesDetailCollectionViewDataSource<U, V: BothamViewCell where U == V.ItemType> : BothamCollectionViewDataSource<U, V> {
```

Comment:
Because you already know the type of this datasource you don't need to keep it generic you can keep the genericity up to this class with the following signature.
class SeriesDetailCollectionViewDataSource : BothamCollectionViewDataSource<Comic, ComicCollectionViewCell>